### PR TITLE
Fix npm i high severity vulnerability

### PR DESCRIPTION
**Pull request by elenavillamil**
This PR is to update webpack dependency to fix npm audit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "2.0.120230310",
+    "version": "2.0.120230321-vscodedependentboot",
```
Comment (Tatsinnit) [inline package.json:5]: 💡☕️ Thanks for this, please note: I do not see any update on the `package.json` file yet the `package-lock` file has many. Essentially, `package-lock` file is the automatically generated file. here is more for read: https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json

**Review by Tatsinnit — COMMENTED** [review]
❤️☕️ Thank you for adding to the review, added some comment, might be `package.json` need all the new package version update, as I can only see `package-lock` file updated but original `package.json` only has version update.

Thanks.

File: package.json (hunk 1 above)
Comment (elenavillamil) [reply package.json:5]: Ah this was leftover from testing. Need to remove it, thanks.

**Comment by Tatsinnit** [comment]
🏛️ 🥷  Adding bit more context on my `dependency` -->  `package.json` comment vs `package-lock.json` its the dependency packages which are not directly mentioned in package.json are getting updated, for example `chownr` - depends on `prebuilt-install` - depends on `vsce` - So I wonder if it makes sense to bring the `package.json` to latest packages which will take care of dependent packages? something in lines of what this guys is trying to say: https://stackoverflow.com/a/74641973/1210059  


Screenshot 2023-03-22 at 1 16 28 PM

#### Thoughts share: 💡 Might be we can do : `npm outdated`  and updated direct dependencies to latest.


Screenshot 2023-03-22 at 1 32 42 PM

**Comment by elenavillamil** [comment]
I am opened to suggestions. This PR is from running npm audit fix. It just touched package-lock and not package.json. This will fix the high severity so extension is compliant. Have you ran outdated command locally and compile, does it introduce breaking changes?

**Review by hsubramanianaks — COMMENTED** [review]
Thanks for this PR. @elenavillamil if you see dependantbot has created a [PR ](https://github.com/Azure/vscode-bridge-to-kubernetes/pull/37/files)for same issue which is updating both files because of whatever @Tatsinnit explained in his comments. (due to transitive dependency). your PR looks good, but the tricky part is package.json still has the webpack as ```"webpack": "^5.75.0",``` ```^``` means only minor version updates (refer semver link for npm below). package-lock.json locks it to any version and easily someone can revert it back because we cannot review huge changes from lock files always (it is hard to follow). so, it is always better to update version for direct dependency in package.json like the dependant bot it. below are some references.
**NOTE: for some reason npm audit fix didn't update this file, not sure why.**

- https://semver.npmjs.com/
- suggestion: it is better to update it as ```webpack": ">=5.76.0"``` so always version greater than  ```5.76.0``` is pulled and correct package-lock.json is generated. 
- another suggestion would be like this ex: ```^5.76.0``` similar to dependant bot.

image

**Comment by elenavillamil** [comment]
I updated package.json manually since audit fix wasnt updating. I did not run npm outdated since i felt that was a bit out of scope for this change.

**Review by hsubramanianaks — APPROVED** [review]
looks good to me.

**Review by hsubramanianaks — CHANGES_REQUESTED** [review]
CLI version is wrong , it should be 1.0.20230327.2 instead of 1.0.20230310.3

**Comment by hsubramanianaks** [comment]
Since this is from Elena's fork I couldn't edit them in the same PR, closing this and making changes from my forked branch to complete this PR.

**Pull request closed without merging**